build.rs

```diff
@@ -146,11 +148,11 @@ fn parse_version(version: &str) {
             );
         }
         println!("cargo:rustc-cfg=mysql_5_7_x");
-    } else if version.starts_with("8.0.") || version.starts_with("21.") {
+    } else if version.starts_with("8.0.") || version == "21" || version.starts_with("21.") {
```

Comment:
Cann you extend the comment in line 135-140 to explain why we use `version == "2x"` here?